File: src/Task.java
public class Task {

    // Private Fields
    
    private String taskType;
    private String[] taskTypes = {"Classroom Support Call", "Lab Support", "Housekeeping"};
    private String taskStatus;
    private String[] taskStatuses = {"Ongoing", "Resolved"};
    private String location;
    private String issue;
    private String action;
    private int incidentID;

    // Public Constructor

    public Task(int taskTypeNumber, String location, String issue, String action, int taskStatusNumber, int incidentID) {
        this.taskType = taskTypes[taskTypeNumber];
        this.location = location;
        this.issue = issue;
        this.action = action;
        this.taskStatus = taskStatuses[taskStatusNumber];
        this.incidentID = incidentID;
    }
    
    // Getters/Setters
    public String getTaskType() {
        return taskType;
    }
    public String getTaskStatus() {
        return taskStatus;
    }
    public String getLocation() {
        return location;
    }
    public String getIssue() {
        return issue;
    }
    public String getAction() {
        return action;
    }
    public int getIncidentID() {
        return incidentID;
    }
    public void setTaskType(int taskTypeNumber) {
        this.taskType = taskTypes[taskTypeNumber];
    }
    public void setTaskStatus(int taskStatusNumber) {
        this.taskStatus = taskStatuses[taskStatusNumber];
    }
    public void setLocation(String location) {
        this.location = location;
    }
    public void setIssue(String issue) {
        this.issue = issue;
    }
    public void setAction(String action) {
        this.action = action;
    }
    public void setIncidentID(int incidentID) {
        this.incidentID = incidentID;
    }

    
}

File: task.py
class Task:
    def __init__(self, taskType, ticketNumber, location, issue, action, status):
        self.taskType = taskType
        self.ticketNumber = ticketNumber
        self.localtion = location
        self.issue = issue
        self.action = action
        self.status = status

    # Setters
    def setType(self, newType):
        self.taskType = newType
    def setTicketNumber(self, newTicketNumber):
        self.ticketNumber = newTicketNumber
    def setLocation(self, newLocation):
        self.location = newLocation
    def setIssue(self, newIssue):
        self.issue = newIssue
    def setAction(self, newAction):
        self.action = newAction
    def setStatus(self, newStatus):
        self.status = newStatus

    # Getters
    def getType(self):
        return self.taskType
    def getTicketNumber(self):
        return self.ticketNumber
    def getLocation(self):
        return self.location
    def getIssue(self):
        return self.issue
    def getAction(self):
        return self.action
    def getStatus(self):
        return self.status
